### PR TITLE
Csv manipulations

### PR DESCRIPTION
**Pull request by bhavesha-systango**

### CODE REVIEW

File: day6/salestax.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import csv
+import pandas
+import fs as fs
```
Comment (araghuvanshi-systango) [inline day6/salestax.py]: why `import fs as fs`? you don't need `as fs` here as it is already imported as `fs`

File: day6/salestax.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+csv_rowlist = [["ProductName","ProductCost","Country"], ["ponds",150,"india"],
+               ["boro-plus",200,"portuguese"],["himalaya",500,"spain"]]
+with open('users.csv', 'w') as file:
+    writer = csv.writer(file)
+    writer.writerows(csv_rowlist)
+
+with open('users.csv', 'r',) as file:
```
Comment (araghuvanshi-systango) [inline day6/salestax.py]: there is comma after `'r'`.

File: day6/salestax.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+with open("output10.csv", "w") as file:
+    dfd = repr(df)
+    file.write(dfd + "\n")
+
+
+
+
+
```
Comment (araghuvanshi-systango) [inline day6/salestax.py]: Too much spaces at the end of file.

File: day7/salestax2.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import csv
+import pandas
+import fs as fs
+if __name__=='__main__':
```
Comment (araghuvanshi-systango) [inline day7/salestax2.py:6]: Give two space after you import finsih

File: day7/salestax2.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+c1.write_file()
+c1.read_file()
+data = c1.calculate_final_price()
+c1.output_to_file(data)
+
+
+
+
+
```
Comment (araghuvanshi-systango) [inline day7/salestax2.py:49]: too many at the end of file

**Review by araghuvanshi-systango — CHANGES_REQUESTED** [review]

File: day6/salestax.py (hunk 1 above)
Comment (bhavesha-systango) [reply day6/salestax.py]: okay

File: day6/salestax.py (hunk 2 above)
Comment (bhavesha-systango) [reply day6/salestax.py]: okay

File: day6/salestax.py (hunk 3 above)
Comment (bhavesha-systango) [reply day6/salestax.py]: okay

File: day7/salestax2.py (hunk 4 above)
Comment (bhavesha-systango) [reply day7/salestax2.py:6]: okay

File: day7/salestax2.py (hunk 5 above)
Comment (bhavesha-systango) [reply day7/salestax2.py:49]: okay

**Pull request merged**